### ISSUE TITLE
Logs don't appear when running a worker

### ISSUE THREAD

**Issue by markoa** [heroku/toolbelt#148]
Here's what I have:

`Procfile`:
```
worker: bundle exec ruby test.rb
```

`test.rb`:
```
puts "Hello"
puts "Hola"
sleep(1000)
```

Running `heroku local --port 4000` gives:

```
forego  | starting worker.1 on port 4000
^C      | ctrl-c detected
forego  | sending SIGTERM to worker.1
worker.1 | Hello
```

I don't quite understand why stdout things don't appear?

**Comment by ransombriggs** [comment]
@markoa There are currently some open issues in the program that `heroku local` uses behind the scenes to run your application.  We are in the process of moving to another program that should fix issues like yours, I will let you know when the fix has been released.

**Comment by markoa** [comment]
Thanks @ransombriggs, I'm looking forward to it.

**Comment by ruffin--** [comment]
Getting a similar issue running `heroku local web` with a node.js app. When run with node directly, I get all the console logging I'd expect, but with `heroku local web`, it stops logging early. Return values still work, and service is still running. Hitting ctrl-c when this happens gives me `ctrl-c detected`, but the app doesn't stop.

Assuming it's a similar thing.

**Comment by ransombriggs** [comment]
@ruffin-- yeah, it is the same issue, will let you know when we have a fix ready

**Comment by ransombriggs** [comment]
@ruffin-- Since you are running a node app, I assume you have experience with using npm.  Would it be possible to help test our `heroku local` fix?  Currently the CLI uses forego under the covers of `heroku local`, but we are investigating moving to [node-foreman](http://strongloop.github.io/node-foreman/) because of issues like you have encountered with heroku local. If you want to test out node-foreman and see if it fixes your problem, that would be very helpful information for us. I have tested it with a simple node app and found it worked well, just run `nf start` instead of `heroku local`.
